Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml

```diff
@@ -75,10 +75,11 @@
                                 Text="{DynamicResource flowlauncher_plugin_websearch_guide_1}"
                                 TextAlignment="Left"
                                 TextWrapping="WrapWithOverflow" />
-                            <TextBlock
+                            <TextBox
```

Comment:
what's this for?

Reply:
for selectable text. if user want to test, user can simply drag the url.

Reply:
I didn't get the idea? Isn't Textblock also selectable?

Reply:
normal textblock can't selectable. (maybe there are way but I didn't checked.) but textbox more look like selectable. if you think looks weired. I'll change it.  😘

Reply:
Na it's OK. i am just curious about what it is for.